ddtrace/settings/_inferred_base_service.py

```diff
@@ -0,0 +1,203 @@
+import fnmatch
+import os
+import pathlib
+import re
+import sys
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+
+INIT_PY = "__init__.py"
+ALL_PY_FILES = "*.py"
+
+CACHE: Dict[str, Optional[str]] = {}
+
+
+class DetectionContext:
+    def __init__(self, envs):
+        self.envs = envs
+
+
+class ServiceMetadata:
+    def __init__(self, name):
+        # prevent service name from being an empty string
+        self.name = name if name != "" else None
+
+
+class PythonDetector:
+    def __init__(self, ctx):
+        self.ctx = ctx
+        name = "python"
+        self.name = name
```

Comment:
```suggestion
        self.name = "python"
```

Reply:
`self.name` doesn't appear to be used anywhere? any reason to not just hard code into the pattern?